pss/crypto/crypto.go

```diff
@@ -486,13 +499,18 @@ func (crypto *defaultCryptoBackend) signHash(hash []byte, prv *ecdsa.PrivateKey)
 
 // Obtain public key from the signed message and the signature
 func (crypto *defaultCryptoBackend) sigToPub(signed, sig []byte) (*ecdsa.PublicKey, error) {
+	defer func() { recover() }() // in case of invalid signature
 	hash := crypto.keccak256(signed)
 	return ethCrypto.SigToPub(hash, sig)
 }
 
+func (crypto *defaultCryptoBackend) importECDSAPublic(key *ecdsa.PublicKey) *ecies.PublicKey {
```

Comment:
where is this needed? the naming is confusing

Reply:
Whisper encryption uses ecies (Elliptic Curve Integrated Encryption Scheme) package. This function converts from ecdsa public key to ecies public key. Maybe convert to ECIESPublic is a better name.

Reply:
`ECDSAtoECIESPublic` ?

Reply:
I will go with `fromECDSAtoECIESPublic` so it doesn't start with capitals.